src/modules/tests/test-data/manoeuvres/manoeuvres.cat-be.reducer.ts

```diff
@@ -0,0 +1,13 @@
+import { Manoeuvres } from '@dvsa/mes-test-schema/categories/BE';
+import { Action } from '@ngrx/store';
+
+export const initialState: Manoeuvres = {
+  reverseLeft: {},
+};
+
+export function manoeuvresCatBEReducer(state: Manoeuvres = initialState, action: Action) {
```

Comment:
missing return type and should be Cat Be not Cat BE

Reply:
added it, changed it